CA_DataUploaderLib/ServerUploader.cs

```diff
@@ -471,7 +477,7 @@ async Task<bool> Post()
 
             foreach (var e in events)
             {
-                if (TrackDuplicate(e.Data)) continue;
+                if (TrackDuplicate(e.Data) || (_muteErrors && e.Data.StartsWith("Failed sharing decision"))) continue;

```

Comment:
The check should come from an extension point, instead of a hardcoded message check.

Reply:
done